captum/attr/_core/lrp.py

```diff
@@ -273,38 +308,43 @@ def _check_rules(self):
                         )
                     )
 
-    def _register_forward_hooks(self):
+    def _register_forward_hooks(self) -> None:
         for layer in self.layers:
             if type(layer) in SUPPORTED_NON_LINEAR_LAYERS:
                 backward_handle = layer.register_backward_hook(
                     PropagationRule.backward_hook_activation
                 )
                 self.backward_handles.append(backward_handle)
             else:
-                forward_handle = layer.register_forward_hook(layer.rule.forward_hook)
+                forward_handle = layer.register_forward_hook(
+                    layer.rule.forward_hook  # type: ignore
+                )
                 self.forward_handles.append(forward_handle)
                 if self.verbose:
                     print(f"Applied {layer.rule} on layer {layer}")
 
-    def _register_weight_hooks(self):
+    def _register_weight_hooks(self) -> None:
         for layer in self.layers:
             if layer.rule is not None:
                 forward_handle = layer.register_forward_hook(
-                    layer.rule.forward_hook_weights
+                    layer.rule.forward_hook_weights  # type: ignore
                 )
                 self.forward_handles.append(forward_handle)
 
-    def _register_pre_hooks(self):
+    def _register_pre_hooks(self) -> None:
         for layer in self.layers:
             if layer.rule is not None:
                 forward_handle = layer.register_forward_pre_hook(
-                    layer.rule.forward_pre_hook_activations
+                    layer.rule.forward_pre_hook_activations  # type: ignore
                 )
                 self.forward_handles.append(forward_handle)
 
     def _compute_output_and_change_weights(
-        self, inputs, target, additional_forward_args
-    ):
+        self,
+        inputs: TensorOrTupleOfTensorsGeneric,
```

Comment:
nit: This is actually always Tuple[Tensor, ...] since inputs gets formatted to a tuple in _format_inputs